apps/web/src/components/GalleryEditor/PiecesSidebar/PiecesSidebar.tsx

```diff
@@ -199,17 +206,24 @@ export function PiecesSidebar({ tokensRef, queryRef }: Props) {
     setSelectedWallet('All');
   }, []);
 
+  const availableChains = chains
+    .filter((chain) => chain.name !== 'All Networks')
+    .map((chain) => chain.name as Exclude<(typeof chains)[number]['name'], 'All Networks'>);
```

Comment:
love this 😍 

1. let's make sure to `useMemo` this array
2. i can see this type `Exclude<(typeof chains)[number]['name'], 'All Networks'>` being exported from `chain.ts` as it seems useful. can be `export type AvailableChainsExceptAllNetworks`